account-kit/react/src/components/auth/card/loading/email.tsx

```diff
@@ -7,10 +7,10 @@ import { useAuthContext, type AuthStep } from "../../context.js";
 import { Spinner } from "../../../../icons/spinner.js";
 import { ls } from "../../../../strings.js";
 import { EmailIllustration } from "../../../../icons/illustrations/email.js";
-import type { AuthCardProps } from "../index.js";
+import type { AlchemyAccountsUIConfig } from "../../../../context.js";
 
 interface LoadingEmailProps {
-  config: AuthCardProps;
+  config: AlchemyAccountsUIConfig;
```

Comment:
here and below, similar comment as the `AddPasskey` component. We should be able to just pull this form the `useAuthContext()` hook now right? or is there still a step that relies on this?